nodes/KlickTipp/actions/field/get.operaton.ts

```diff
@@ -0,0 +1,46 @@
+import type { IExecuteFunctions, INodeProperties } from 'n8n-workflow';
+import { apiRequest } from '../../transport';
+import { handleError, handleObjectResponse, updateDisplayOptions } from '../../utils/utilities';
+
+export const properties: INodeProperties[] = [
+  {
+    // eslint-disable-next-line n8n-nodes-base/node-param-display-name-wrong-for-dynamic-options
+    displayName: 'Field ID',
+    name: 'fieldId',
+    type: 'options',
+    typeOptions: {
+      loadOptionsMethod: 'getFields',
+    },
+    default: '',
+    options: [],
+    description:
+      'Choose from the list, or specify an ID using an <a href="https://docs.n8n.io/code-examples/expressions/">expression</a>',
+  },
+];
+
+const displayOptions = {
+  show: {
+    resource: ['field'],
+    operation: ['get'],
+  },
+};
+
+export const description = updateDisplayOptions(displayOptions, properties);
+
+export async function execute(this: IExecuteFunctions, index: number) {
+  const fieldId = this.getNodeParameter('fieldId', index) as string;
+
+  //Extract just field name, i.e. CompanyName
+  const fieldName = fieldId.replace(/^field/, '');
```

Comment:
nit: Correctly this should be called fieldApiId and fieldId.
The name is an extra property. At least we call it FieldName in the backend. But could be that the nameing in the API you use is not exactly the same => choose the best matching name
This is an example from an API call form the app Frontend (which does not represent the same object as the legacy API)
![image](https://github.com/user-attachments/assets/0cf33163-3a02-4f9d-87be-fb0c7fc5a262)

Reply:
Fixed, I also duplicated these changes in Node-RED app